### ISSUE TITLE
Q: Status / roadmap for raid5/6 support?

### ISSUE THREAD

**Issue by mabod** [btrfs/btrfs-todo#66]
I was searching the web for information about current status of RAID5/6 support in btrfs, but could not find anything. 

RAID5/6 is not approved for production use since many years. I could not find any signs for progress on this topic.  And I could not find a roadmap or an outlook. So here I am, addressing the question to the developers.

Is RAID5/6 support still  a goal for btrfs? And if so, do you have a roadmap?  And if not, would it be more honest to drop RAID5/6 support for good and tell the people that it is not in scope anymore?

**Comment by kdave** [comment]
We don't have a formal road map. If you search again in the mailinglists you'd find mentions of RAID56 that will be fixed by feature in development *raid-stripe-tree*, that's mainly for zoned mode but also addresses the problem with write hole of RAID56, basically adding another layer of extent tracking.

So once RST is finished it'll also fix the RAID56 problems. Dropping it was suggested but fixing it is preferrable.

**Comment by kdave** [comment]
Also please don't open issues in this repository, it's for patch tracking and I noticed this issue incidentally.

**Comment by Renha** [comment]
In what repository are we supposed to open issues?
